app/src/main/java/dev/msfjarvis/aps/ui/git/base/BaseGitActivity.kt

```diff
@@ -115,6 +103,26 @@ abstract class BaseGitActivity : ContinuationContainerActivity() {
         }
     }
 
+    /**
+     * Takes the result of [launchGitOperation] and applies any necessary transformations
+     * on the [throwable] returned from it
+     */
+    private fun transformGitError(throwable: Throwable): Throwable {
+        val err = rootCauseException(throwable)
+        return if (err.message?.contains("cannot open additional channels") == true) {
```

Comment:
This could be a `when`, but LGTM apart from that.

Reply:
Yeah the `when` looks less 'busy', made the change.